### PR TITLE
package: bump hogan for lambda same subs fix

### PR DESCRIPTION
**Pull request by nemtsov**
Diff: https://github.com/behance/hogan.js/compare/v3.0.6...behance:v3.0.7

### CODE REVIEW

**Comment by matt-oconnell** [comment]
LGTM

File: package.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
     "nodemon": "^1.11.0"
   },
   "dependencies": {
-    "hogan.js": "github:behance/hogan.js.git#v3.0.6",
+    "hogan.js": "github:behance/hogan.js.git#v3.0.7",
```
Comment (weikinhuang) [inline package.json]: why the `github:` prefix

Comment (nemtsov) [inline package.json]: It was required by an older version of npm. Not needed now. https://docs.npmjs.com/files/package.json#dependencies

Comment (weikinhuang) [inline package.json]: gotcha

Comment (nemtsov) [inline package.json]: fixed.

**Pull request merged**